pkg/sql/tablemetadatacache/util/util.go

```diff
@@ -14,3 +14,12 @@ import "context"
 type ITableMetadataUpdater interface {
 	RunUpdater(ctx context.Context) error
 }
+
+// NoopUpdater is an implementation of ITableMetadataUpdater that performs a noop when RunUpdater is called.
```

Comment:
This should only be used in tests right? Maybe note that.

Reply:
Done